dist/_patterns/01-components/01-page-elements/hero.twig

```diff
@@ -0,0 +1,10 @@
+<div class="hero">
+  <div class="hero__content" role=”banner”>
+    <h1 class="hero__header">{{ pageTitle }}</h1>
+    <h2 class="hero__subheader">{{ pageSubTitle }}</h2>
+    <p class="hero__text">{{ summary }}</p>
```

Comment:
Are heros always connected to content on a site? Perhaps it shouldn't be summary but field content.